### PR TITLE
[TS-180] Cypress testing for trending companies component

### PR DESCRIPTION
**Pull request by rileykc98**

### CODE REVIEW

**Review by AlyaNaseer — APPROVED** [review]
Passes locally, no code smells and follows naming conventions. Great work!

File: cypress/integration/E2E_tests/TradeZoneTests/trendingCompanies.spec.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+describe("On the tradezone page, TrendingCompanies component can", () => {
+  before(() => {
+    cy.login();
+    cy.wait(2000);
+    cy.fixture("url").then((jsonFile) => {
+      cy.visit(
+        `${jsonFile.HOMEPAGE}:${process.env.PORT || 8081}${jsonFile.TRADEZONE}`
+      );
+    });
+  });
```
Comment (kennguyen0303) [inline cypress/integration/E2E_tests/TradeZoneTests/trendingCompanies.spec.js:10]: Looks good

**Review by kennguyen0303 — APPROVED** [review]
Looks good. 

I like the fact that you put login and visit the page to `before(..)`; looks very clean.

File: cypress/integration/E2E_tests/TradeZoneTests/trendingCompanies.spec.js (hunk 1 above)
Comment (rileykc98) [reply cypress/integration/E2E_tests/TradeZoneTests/trendingCompanies.spec.js:10]: Thanks!

**Pull request merged**